solidity/contracts/libs/OPL2ToL1Metadata.sol

```diff
@@ -31,9 +31,9 @@ library OPL2ToL1Metadata {
     //      _data
     //          OFFSET                      = 32 bytes
     //          LENGTH                      = 32 bytes
-    //          PADDING + verifyMessageId   = 64 bytes
-    // } = 292 bytes
-    uint256 private constant MESSENGER_CALLDATA_LENGTH = 292;
+    //          PADDING + verifyMessageId   = 96 bytes
+    // } = 324 bytes
+    uint256 private constant MESSENGER_CALLDATA_LENGTH = 324;
```

Comment:
you can include constant expressions here for clarity
something like
```
    uint256 private constant MESSENGER_CALLDATA_LENGTH = 4 + 8 * 32;
```
although this is 260...

Reply:
I think you're missing the offset and length.